src/main/java/org/folio/rest/impl/CirculationSettingsAPI.java

```diff
@@ -21,7 +27,10 @@ public void postCirculationSettingsStorageCirculationSettings(String lang,
 
     new CirculationSettingsService(vertxContext, okapiHeaders)
       .create(circulationSettings)
-      .onComplete(asyncResultHandler);
+      .onSuccess(reply -> asyncResultHandler.handle(
```

Comment:
I think it's Handler<Response>, so it's better to rename it to "response -> " or "r -> "

Reply:
done